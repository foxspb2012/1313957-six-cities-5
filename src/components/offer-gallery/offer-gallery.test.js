import React from 'react';
import renderer from 'react-test-renderer';
import OfferGallery from './offer-gallery';

const PHOTOS = [`img/apartment-01.jpg`, `img/apartment-03.jpg`];

describe(`Should OfferGallery render correctly`, () => {
  it(`Should OfferGallery render correctly with photos`, () => {
    const tree = renderer
      .create(
          <OfferGallery images={PHOTOS} />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`Should OfferGallery render correctly without photos`, () => {
    const tree = renderer
      .create(
          <OfferGallery />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
