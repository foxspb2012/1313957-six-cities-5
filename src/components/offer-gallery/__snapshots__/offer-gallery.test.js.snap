// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should OfferGallery render correctly Should OfferGallery render correctly with photos 1`] = `
<div
  className="property__gallery-container container"
>
  <div
    className="property__gallery"
  />
</div>
`;

exports[`Should OfferGallery render correctly Should OfferGallery render correctly without photos 1`] = `
<div
  className="property__gallery-container container"
>
  <div
    className="property__gallery"
  />
</div>
`;
